test/config.js

```diff
@@ -76,8 +76,8 @@ describe('embark.Config', function () {
         "genesisBlock": "config/development/genesis.json",
         "datadir": ".embark/development/datadir",
         "isDev": false,
-        "targetGasLimit": "20000000",
-        "gasPrice": "8000000",
+        "targetGasLimit": "0x1312d00",
+        "gasPrice": "0x7a1200",
```

Comment:
Okay I think I caused something here that wasn't intended. The values should still be the number strings. So while the util function still returns hex values in case a unitless value is given, we still need to convert those to number strings (just not within `getWeiBalanceFromString`, or if we do it within, we should make this configurable)

Reply:
@PascalPrecht Ok, I think I understand - `getWeiBalanceFromString` needs to retain its original behavior, but the config object should only contain number strings. Sorry for the confusion! 

I've added a commit (https://github.com/embark-framework/embark/pull/910/commits/786e1c7a513323555a70f6128226938fffafbdde) that takes a simpler approach to effect the desired behavior by replacing the `'auto'` string check with a simple regex check for the unit string, which also ignores the `'auto'` string. Sorry for changing tactics at this point but I think this both simplifies the changes in this branch as well as directly trigger the unit string conversion in a more understandable way. Let me know what you think and thanks for the help!